test/provider_test.exs

```diff
@@ -181,17 +70,17 @@ defmodule ProviderTest do
       System.put_env("OPT_6", "false")
       System.put_env("OPT_7", "3.14")
 
+      {:ok, pid} = TestModule.start_link()
+
       assert TestModule.opt_1() == "some data"
       assert TestModule.opt_2() == 42
       assert TestModule.opt_3() == "foo"
       assert TestModule.opt_4() == "bar"
       assert TestModule.opt_5() == "baz"
       assert TestModule.opt_6() == false
       assert TestModule.opt_7() == 3.14
-    end
 
-    test "access function raises for on error" do
```

Comment:
This test is no longer applicable since we are expecting someone to call `MyConfig.start_link()` before attempting to use the values. And if we did that before the test like we did in other places, we would get an error at that point.